README.md

```diff
@@ -15,14 +15,12 @@
 
 [Live Demo Link](https://livelink)
```

Comment:
- [ ] As requested by the previous code reviewer, kindly replace the live link you have added with a working one or remove it completely as it is not a requirement.